frontend/lib/home/app_flows_stack.dart

```diff
@@ -10,16 +10,9 @@ class AppFlowsStack extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return WillPopScope(
-      onWillPop: () async {
-        final currentState = appFlows[currentIndex].navigatorKey.currentState;
-        final currentStatePopped = await currentState?.maybePop() ?? false;
-        return !currentStatePopped;
-      },
-      child: IndexedStack(
-        index: currentIndex,
-        children: _buildChildren(),
-      ),
+    return IndexedStack(
```

Comment:
@michael-markl or @maxammann i tried to find out why we need this and read some documentation when to use and couldn't find a proper use case why this is required. Can you give me a hint? `WillPopScope` is deprecated so i would have to update the widget but it seems like everything is working without that widget as far as i can see

Reply:
I think this might have been necessary to avoid closing the app (via e.g. Android back button), when the user actually only wants to dismiss a detail page in one of the tabs. As we open the screens in the root navigator by now, the WillPopScope might no longer be necessary (Maybe it still is for dialogs, maybe check what the Android back button does when the Map Contributors dialog is shown?)...

Reply:
if the contributors dialog is open and i push back button it closes the modal. so it should be fine